### ISSUE TITLE
Event Request OnBeforeOnRun in Codeunit 5641 "FA Reclass. Check Line"

### ISSUE THREAD

**Issue by ReneZiel** [microsoft/ALAppExtensions#21097]
Hello, please consider adding the following event to Codeunit 5641 "FA Reclass. Check Line"

```
[IntegrationEvent(false, false)]
    local procedure OnBeforeOnRun(var FAReclassJournalLine: Record "FA Reclass. Journal Line"; var IsHandled: Boolean)
    begin
    end;
```
It should be call at first in the OnRun trigger.

```
trigger OnRun()
    var
      IsHandled: Boolean;
    begin
        IsHandled := false;
        OnBeforeOnRun(rec,IsHandled);
        if IsHandled then
            exit;
```
many thanks!

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 49832.

**Comment by ReneZiel** [comment]
Guten Morgen Karl,

dein Event ist in der nächsten BC Version schon drin.

​​​​​​​Mit freundlichem Gruß

René Zielinski
Softwareentwickler
Bereich Digitalisierung & Plattformen > Microsoft Business Central Plattform - SI​


***@***.***

Axians Infoma GmbH
EASTSITE VII / Hermsheimer Str. 3 - 68163 Mannheim - Deutschland​
Tel.: +49 621 72926-172 - Mobil: +49 1512 4111889 - Fax: +49 621 72926-200
***@***.******@***.***>​
www.axians-infoma.de<https://www.axians-infoma.de/>

Amtsgericht Ulm, Handelsregister-Nummer: HRB 1961
Geschäftsführer: Daniel Riss, Holger Schmelzeisen


***@***.***<https://eur01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.vinci-energies.com%2F&data=05%7C01%7Crene.zielinski%40axians-infoma.de%7C0442dbb2b99e42194bde08dabd8afade%7Ccae7d06108f340dd80c33c0b8889224a%7C0%7C0%7C638030712168604967%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=YaN5XcjLnLZK8GbvMq8c%2FBeh%2BD%2B1gJpJ4g8SbUjYyDk%3D&reserved=0>​​​
***@***.***<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.axians-infoma.de%2Fservices%2Fnewsletter%2F&data=05%7C01%7Crene.zielinski%40axians-infoma.de%7C0442dbb2b99e42194bde08dabd8afade%7Ccae7d06108f340dd80c33c0b8889224a%7C0%7C0%7C638030712168604967%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xifUiwMNKinqX2Ng6MGJimamGqrTjNT7QXTxGeoKInc%3D&reserved=0>



Von: Jesper Schulz-Wedde ***@***.***>
Gesendet: Mittwoch, 23. November 2022 18:50
An: microsoft/ALAppExtensions ***@***.***>
Cc: ZIELINSKI Rene ***@***.***>; Author ***@***.***>
Betreff: Re: [microsoft/ALAppExtensions] Event Request OnBeforeOnRun in Codeunit 5641 "FA Reclass. Check Line" (Issue #21097)


Closed #21097<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FALAppExtensions%2Fissues%2F21097&data=05%7C01%7Crene.zielinski%40axians-infoma.de%7C92795ec145764295ee9e08dacd7b1e52%7Ccae7d06108f340dd80c33c0b8889224a%7C0%7C0%7C638048225923079143%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GJ8vMD8ny8h%2FyyZlpzffdQvFKdcJKe2Bhk1Pz0NKevw%3D&reserved=0> as completed.

—
Reply to this email directly, view it on GitHub<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FALAppExtensions%2Fissues%2F21097%23event-7879518225&data=05%7C01%7Crene.zielinski%40axians-infoma.de%7C92795ec145764295ee9e08dacd7b1e52%7Ccae7d06108f340dd80c33c0b8889224a%7C0%7C0%7C638048225923079143%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2F0Ntx%2B1XygtxZ6DPoCgXAPcZtOX1DW7KLgg%2FfiSk%2Fhw%3D&reserved=0>, or unsubscribe<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAYAWGE3LUMFGFG7X4DTSGETWJZKLZANCNFSM6AAAAAASGSX2AU&data=05%7C01%7Crene.zielinski%40axians-infoma.de%7C92795ec145764295ee9e08dacd7b1e52%7Ccae7d06108f340dd80c33c0b8889224a%7C0%7C0%7C638048225923079143%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nboTlzV7aO07bPPNVrkkBDJU2ZZndRaEZFdvc4TDN%2BY%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

Axians Infoma GmbH | Hörvelsinger Weg 21 | D-89081 Ulm | Tel: +49 731 1551-0 | www.axians-infoma.de | Amtsgericht Ulm | HRB 1961 | Geschäftsführer: Daniel Riss, Holger Schmelzeisen

Der Inhalt dieser E-Mail einschließlich der Anlagen ist vertraulich und unter Umständen rechtlich geschützt. Der Inhalt ist ausschließlich an den eigentlich vorgesehenen Empfänger gerichtet. Sollten Sie diese E-Mail irrtümlich erhalten haben, benachrichtigen Sie den Absender bitte unverzüglich per Antwort-E-Mail, und löschen Sie diese Nachricht einschließlich der Anlagen.

This electronic message, including any attachment, contains confidential information and may be privileged or otherwise be protected from disclosure. The information is intended to be for the use of the intended addresses only. If you have received this message by mistake, please notify the sender immediately by reply e-mail, and delete this message and any attachment from your system.
